tiledb/sm/query/result_tile.h

```diff
@@ -271,6 +288,9 @@ class ResultTile {
   /** Maps attribute names to tiles. */
   std::unordered_map<std::string, TileTuple> attr_tiles_;
 
+  /** Maps attribute names to size ratios */
+  std::unordered_map<std::string, double> query_size_ratios_;
```

Comment:
could we map directly to the new size rather than a ratio? i want to avoid the complexity of a floating point

Reply:
I will change those map to std::unordered_map<Datatype, Datatype> >